example/lib/main.dart

```diff
@@ -240,24 +241,27 @@ class Password extends FormzInput<String, PasswordValidationError> {
   }
 }
 
-extension on EmailValidationError {
-  String text() {
+extension on FutureOr<EmailValidationError?> {
```

Comment:
Do we really need this change? I think the extension would also work fine as it was before.